src/web/components/Input/Input.scss

```diff
@@ -25,7 +25,6 @@
 }
 
 .option-label {
-  padding-left: 0.5rem;
```

Comment:
I moved this to right-margin on the checkbox instead so it would work better with custom-provided labels.